Android.mk

```diff
@@ -288,9 +291,14 @@ $(TARGET_LISTFILES_NOBLOBS): $(TARGET_BLOBFREE_ZIP)
 compare-zipfiles: $(TARGET_LISTFILES_BLOBS) $(TARGET_LISTFILES_NOBLOBS)
 	diff -uw $(TARGET_LISTFILES_BLOBS) $(TARGET_LISTFILES_NOBLOBS) || true
 
+# Necessary to ensure the recovery.fstab file name
+# May fail if a device only defines recovery_fstab
+$(TARGET_BLOBFREE_RECOVERY_FSTAB): $(TARGET_RECOVERY_FSTAB)
```

Comment:
I think we might have a race in the build system so on some devices, sometimes, TARGET_RECOVERY_FSTAB is empty and that's why I had put also recovery_fstab. So I think either we find the proper dependency to make sure the TARGET_RECOVERY_FSTAB variable is always good, or we just get both.